reverse_proxy_handler.cc

```diff
@@ -14,6 +14,7 @@
 RequestHandler::Status ReverseProxyHandler::Init(const std::string& uri_prefix, const NginxConfig& config)
 {
 	prefix_ = uri_prefix;
+	protocol_ = "80";
```

Comment:
I guess I should refactor it to be port, cuz that's what it actually is for. Protocol being set to http fails when trying to resolve the connection which was what was causing the 500 internal server error before.